src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -101,10 +101,10 @@ def load_arguments(self, _):
         c.argument('file', options_list=['--file', '-f'], help="The file path to export the container group.")
 
     with self.argument_context('container restart') as c:
-        c.argument('container_group_name', options_list=['--container-group-name', '-n'], help="The name of the container group to restart.")
-    
+        c.argument('container_group_name', options_list=['--name', '-n'], help="The name of the container group to restart.")
+
     with self.argument_context('container stop') as c:
-        c.argument('container_group_name', options_list=['--container-group-name', '-n'], help="The name of the container group to stop.")
+        c.argument('container_group_name', options_list=['--name', '-n'], help="The name of the container group to stop.")
```

Comment:
You could also just register these under `self.argument_context('container')  as c:` to eliminate the duplication.